File: eel-core/src/test/kotlin/io/eels/FieldTest.kt
package io.eels

import io.eels.schema.Field
import io.kotlintest.specs.WordSpec

class FieldTest : WordSpec() {

  init {
    "Column.toLowerCase" should {
      "return new column with lower case name" {
        val col = Field("MyName")
        col.toLowerCase().name shouldBe "myname"
      }
    }
  }
}

File: eel-core/src/test/scala/io/eels/FieldTest.scala
package io.eels

import io.eels.schema.{Field, StringType}
import org.scalatest.{Matchers, WordSpec}

class FieldTest extends WordSpec with Matchers {
  "Field.toLowerCase" should {
    "return new field with lower case name" in {
      val col = Field("MyName", StringType)
      col.toLowerCase().name shouldBe "myname"
    }
  }
}
